### PR TITLE
[Admin][Taxon] Taxon image unique code validation

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Admin/ManagingTaxonsContext.php [hunk 1]
```diff
@@ -67,6 +76,8 @@ public function iWantToCreateANewTaxon()
      */
     public function iWantToModifyATaxon(TaxonInterface $taxon)
     {
+        $this->sharedStorage->set('taxon', $taxon);
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingTaxonsContext.php]: Do we need to set it?

File: src/Sylius/Bundle/CoreBundle/Resources/config/validation.xml [hunk 2]
```diff
@@ -152,4 +152,22 @@
...
+            <constraint name="Valid" />
+        </property>
+    </class>
+
+    <class name="Sylius\Component\Core\Model\TaxonImage">
+        <constraint name="Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity">
+            <option name="fields">
+                <value>code</value>
+                <value>owner</value>
+            </option>
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/validation.xml]: We should also add this constraint on the database level.

Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/validation.xml]: And we already have this constraint: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/TaxonImage.orm.xml#L21

**Comment by michalmarcinkowski** [comment]
Thanks Grzesiu!

**Pull request merged**